modular_skyrat/modules/horrorform/code/true_changeling.dm

```diff
@@ -20,14 +20,14 @@
 	status_flags = CANPUSH
 	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
 	minbodytemp = 0
-	maxHealth = 750 //Very durable
-	health = 500
+	maxHealth = 500 //3 and 3/4ths crewmen worth of health, vaguely.
+	health = 200 
```

Comment:
```suggestion
	health = 300 
```
Seems low

Reply:
I think now that you (finally) removed vent-crawling, it is fine to raise the health again in return.

Reply:
I would, but 200 leaves it 30 shy of being able to tank an entire pitbull default cell, which I think is a good point for something they can swap in and out off